integration/testnet/network.go

```diff
@@ -447,6 +448,9 @@ func (net *FlowNetwork) AddNode(t *testing.T, bootstrapDir string, nodeConf Cont
 			nodeContainer.opts.HealthCheck = testingdock.HealthCheckCustom(healthcheckAccessGRPC(hostPort))
 			net.AccessPorts[ColNodeAPIPort] = hostPort
 
+			nodeContainer.addFlag("access-address", "access_1:9000")
+			nodeContainer.addFlag("qc-contract-address", unittest.ServiceAccountPublicKey.PublicKey.String())
```

Comment:
Account addresses are decoupled from authorized keys in general. We can get the address of the service account on localnet and integration tests with:
```suggestion
			nodeContainer.addFlag("qc-contract-address", flow.Testnet.Chain.ServiceAddress())
```